crates/interpreter/src/instructions/bitwise.rs

```diff
@@ -20,6 +20,31 @@ pub fn gt<WIRE: InterpreterTypes, H: ?Sized>(context: InstructionContext<'_, H,
     *op2 = U256::from(op1 > *op2);
 }
 
+pub fn clz<WIRE: InterpreterTypes, H: ?Sized>(context: InstructionContext<'_, H, WIRE>) {
+    check!(context.interpreter, OSAKA);
+    gas!(context.interpreter, gas::VERYLOW);
+    popn_top!([], op1, context.interpreter);
+
+    // From EIP-7939: Count leading zeros (CLZ) opcode
```

Comment:
all of this is redundant, already happening in .leading_zeros (counting limbs.iter().rev() yields 0 for max and 256 for 0)